gpii/node_modules/flowManager/src/FlowManager.js

```diff
@@ -193,6 +195,12 @@ fluid.defaults("gpii.flowManager.local", {
         },
         userErrors: {
             type: "gpii.userErrors"
+        },
+        gpiiRevisionRequester: {
+            type: "gpii.flowmanager.revisionRequester"
+        },
+        repositorySolutionsLoader: {
```

Comment:
As said above, "repositorySolutionsLoader" could be a subcomponent of "solutionsRegistryDataSource" as it's only needed there.